### PR TITLE
[fix] add datahub in header

### PR DESCRIPTION
**Pull request by fcoronelmakingsense**
- Add datahub in header and selected option
- Add last doppler ui system build to fix header navigation

### CODE REVIEW

**Comment by andresmoschini** [comment]
👍

File: src/headerData.json [hunk 1]
```diff
@@ -108,19 +105,23 @@
...
       "isEnabled":false,
-      "isSelected":true,
+      "isSelected":false,
       "idHTML":"templateMenu",
       "$$hashKey":"object:10"
+    },
+    {
+      "title":"Doppler Data Hub",
+      "url":"https://cdn.fromdoppler.com/doppler-webapp/int-latest/",
+      "isEnabled":false,
```
Comment (cbernat) [inline src/headerData.json]: What's the meaning of this isEnabled always in false for all items?

Comment (fcoronelmakingsense) [reply src/headerData.json]: BE things haha

File: src/components/Header/HeaderUserMenu/HeaderUserMenu.js [hunk 2]
```diff
@@ -9,7 +9,7 @@ const HeaderUserMenu = (props) => {
           {user.avatar.text}
         </span>
       </span>
-      <div class="user-menu">
+      <div className="user-menu">
```
Comment (cbernat) [inline src/components/Header/HeaderUserMenu/HeaderUserMenu.js]: 👍

**Comment by cbernat** [comment]
👍

File: src/headerData.json (hunk 1 above)
Comment (cbernat) [reply src/headerData.json]: If you do not use it, why mapping it here? :)

**Pull request merged**